content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -58,9 +58,9 @@ Meanwhile, on the Istio side you can track the progress in [Issue #40394](https:
 Progress has slowed down a bit lately as we continue working with Envoy on various issues, however, we are hoping to get
 this resolved before Istio 1.17 launches in February.
 
-It is our hope that for Istio 1.17 we are able to get rudimentary support for dual-stack working. Iris Ding (Intel) and `Li Chun (Intel)`
-are already working with the community for getting redirection of network traffic for ambient. We will
-announce what changes we have made in Istio 1.17 shortly after its released. 
+It is our hope that for Istio 1.17 we are able to get rudimentary support for dual-stack working. Iris Ding (Intel) and
```

Comment:
i thought all the changes are in already?

Reply:
Not all of the changes made it. @zhlsunshine what changes are left?

Reply:
Hi @jacob-delgado & @kfaseela,  please refer to https://github.com/istio/istio/pull/41248 for some still need to be done. Such as headless service and service entry.

Reply:
@zh;sunshine @jacob-delgado I believe most of the things are already done, except the above. And 1.17 release is approaching soon. Then please reword the blog wherever you mention "We hope to have things in 1.17". Better to word it as majority of the dual stack support is present in 1.17.. and we merge the blog along with the release. You can quote the above issue in a Known Issues section

Reply:
> Not all of the changes made it. @zhlsunshine what changes are left?

Hi @jacob-delgado, sorry for the misunderstanding. I think all is okay

Reply:
I'm rewriting this section. I'll put up a PR up shortly.